pipeline/vars/lib.groovy

```diff
@@ -215,4 +217,79 @@ def SendUMBMessage(def msgMap, def overrideTopic, def msgType){
 
 }
 
+def sendEmail(def testResults, def artifactDetails, def tierLevel){
+    /*
+        Send an Email
+        Arguments:
+            testResults: map of the test suites and its status
+                Example: testResults = [ "01_deploy": "PASS", "02_object": "PASS"]
+            artifactDetails: Map of artifact details
+                Example: artifactDetails = ["composes": ["rhe-7": "composeurl1",
+                                                         "rhel-8": "composeurl2"],
+                                            "product": "Redhat",
+                                            "version": "RHCEPH-5.0",
+                                            "ceph_version": "16.2.0-117",
+                                            "repository": "reponame"]
+            tierLevel:
+                Example: Tier0, Tier1, CVP..
+    */
+    def status = "STABLE"
+    def toList = "ceph-qe-list@redhat.com"
+    def body = readFile(file: "pipeline/vars/emailable-report.html")
+
+    body += "<body>"
+    body += "<h2><u>Test Artifacts</u></h2>"
+    body += "<table>"
+
+    if (artifactDetails.product){body += "<tr><td>PRODUCT</td><td>${artifactDetails.product}</td></tr>"}
+    if (artifactDetails.version){body += "<tr><td> VERSION </td><td>${artifactDetails.version}</td></tr>"}
+    if (artifactDetails.ceph_version){body += "<tr><td> CEPH-VERSION </td><td>${artifactDetails.ceph_version}</td></tr>"}
+    if (artifactDetails.composes){body += "<tr><td>COMPOSES</td><td>${artifactDetails.composes}</td></tr>"}
+    if (artifactDetails.repository){body += "<tr><td> REPOSITORY </td><td>${artifactDetails.repository}</td></tr>"}
+
```

Comment:
All upper case seems to be loud... could we start with Capital letters and then use small. For example
`PRODUCT -> Product`

Reply:
Also instead of `repository`, could we use `Container Image`